services/API-service/src/api/notification/notification.service.ts

```diff
@@ -114,51 +118,51 @@ export class NotificationService {
     }
   }
 
-  private async sendNotificationsFinishedEvents(
-    countryCodeISO3: string,
-    disasterType: DisasterType,
-    noNotifications: boolean,
-    lastUploadDate: LastUploadDateDto,
-  ): Promise<void | NotificationApiTestResponseChannelDto> {
-    const response = new NotificationApiTestResponseChannelDto();
-    const finishedNotifiableEvents =
-      await this.eventService.getEventsSummaryTriggerFinishedMail(
-        countryCodeISO3,
-        disasterType,
-      );
+  // private async sendNotificationsFinishedEvents(
```

Comment:
NICE TO HAVE: I avoid leaving in commented code when possible. If I need the deleted code again, I check the git history.